examples/configurable/configurable.bal

```diff
@@ -0,0 +1,26 @@
+import ballerina/io;
+
+// Initialize the required `configurable` variables with the `?`.
+// A value must be supplied for these variables in a `configuration.toml` file.
+configurable string hostName = ?;
+configurable int port = ?;
+
+// Initialize the optional `configurable` variables with their default values.
+// The value provided here will be overridden by the value specified in a `configuration.toml` file.
+// The values of the `enableRemote` and `maxPayload` variables are overridden.
+configurable boolean enableRemote = true;
+configurable float maxPayload = 1.0;
+
+// The value `http` is used to initialize the `protocol` variable as it is not provided in the
+// `configuration.toml` file.
```

Comment:
```suggestion
// The value `http` is used to initialize the `protocol` variable since a value is not 
// provided for it in the `configuration.toml` file.
```